javascript/node/selenium-webdriver/testing/index.js

```diff
@@ -46,6 +45,19 @@ const { Browser } = require('../lib/capabilities')
 const { Builder } = require('../index')
 const { getBinaryPaths } = require('../common/driverFinder')
 
+let runfiles
+try {
+  // Attempt to require @bazel/runfiles
+  runfiles = require('@bazel/runfiles').runfiles
+} catch (error) {
+  // Handle error if @bazel/runfiles is not supported by mocha
+  const errorMessage = `Error requiring @bazel/runfiles: ${error.message}\n
+   Note: If you are running tests with Mocha or Jasmine, this module is not needed.\n
+   For more details, see: https://github.com/bazelbuild/rules_nodejs/issues/3770`
+  console.error(errorMessage)
```

Comment:
We don't need to log a message. For most users, this will just be noise they don't need